workspace/src/app/views/pages/Project/FileWidget/FileWidget.tsx

```diff
@@ -93,9 +93,9 @@ export const FileWidget = () => {
                     </CardTitle>
                     <CardOptions>
                         <IconButton
-                            name="item-add-artefact"
+                            name="item-upload"
                             data-test-id="project-files-widget-add-file-btn"
-                            text={t("common.action.AddSmth", { smth: t("widget.FileWidget.file") })}
+                            text={t("common.action.uploadFiles")}
```

Comment:
Why you didn't use the dialog title here, it's already named "Upload file" and has all translations?